src/components/masters/program/ProgramPage.jsx

```diff
@@ -133,6 +133,7 @@ ProgramPage.propTypes = {
     programName: PropTypes.string.isRequired,
     programSlug: PropTypes.string.isRequired,
     programUUID: PropTypes.string.isRequired,
+    programDocuments: PropTypes.object,
```

Comment:
Can we call out what the expected structure of this object is here?

`PropTypes.shape({
  // ...
})`

Reply:
added 👍